<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Repelentes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1311</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>LISTADO CONSULTADO</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>1261</width>
       <height>321</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <column>
      <property name="text">
       <string>Insecto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Material</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>P. Protección</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>% Repelencia</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>% Aterrizaje</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>% Picadura</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>% Concentración</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tiempo Testeo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Articulo</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Atrás</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>370</y>
       <width>185</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Exportar Consulta</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1332</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
